icu4j/build.xml

```diff
@@ -813,17 +801,13 @@
         <condition property="disp.is.java8" value="${java.version}" else="${java.version} - Must be 1.8!">
             <isset property="is.java8"/>
         </condition>
-        <condition property="disp.java7.bootclasspath" value="${java7.bootclasspath}" else="Not Defined!">
```

Comment:
If we decided to use JDK newer than 8 for release build, we need to cross-compile. But bootclasspath option is deprecated in a new version and need a different way to handle it. So this won't be used in future.